test/server/cards/02_SHD/units/ValiantAssaultShip.spec.ts

```diff
@@ -0,0 +1,70 @@
+describe('Valiant Assault Ship', function () {
+    integration(function (contextRef) {
+        describe('Valiant Assault Ship\'s ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        spaceArena: ['valiant-assault-ship']
+                    },
+                    player2: {
+                        spaceArena: [{ card: 'system-patrol-craft', upgrades: ['shield'] }]
+                    }
+                });
+            });
+
+            it('should give a unit +2/+0 for this attack while ignoring the Sentinel', function () {
```

Comment:
nit: let's add something like "when the opponent has more resources" just to make it super explicit. that makes it easier for debugging if someone accidentally breaks this due to a random change and needs to figure out what the test is doing. Same for other test names

Reply:
👍